### DISCUSSION TITLE
PhotoPrism original folder is empty error

### DISCUSSION THREAD

**Discussion by Nutztoyou** [photoprism/photoprism#4362] [General]
Hi Guys

I'm new to nas and this way of installing apps. Until now, I've only ever known windows.

I've installed ProtoPrism on a synology nas using an online tutorial.

My existing photos on the nas are in a shared folder called pdrivecopy. I then have a sub folder called photos. In that sub folder there are other subfolders called camera, family, work etc where the photos are actually

During the instal, I used the file path to the directory below

docker run -d --name=PhotoPrism \
-p 2342:2342 \
-e PHOTOPRISM_UPLOAD_NSFW=true \
-e PHOTOPRISM_UID=1026 \
-e PHOTOPRISM_GID=100 \
-e TZ=Australia/Sydney \
-e PHOTOPRISM_ADMIN_USER=xxxxxx \
-e PHOTOPRISM_ADMIN_PASSWORD=yyyyyyyyyyyyyy \
-v /volume1/pdrivecopy/photos:/photoprism/storage \
--security-opt seccomp=unconfined \
--security-opt apparmor=unconfined \
photoprism/photoprism

After the instal, when trying to run index it says the folder is empty. Can anyone correct the file path, or is there some other error.  Just for clarity, I don't actually have a folder called **photoprism/storage** but I assume it is required in the path.

Any help would be appreciated.

**Comment by graciousgrey** [comment]
Hello :wave: 

It looks like you have mounted your photo folder as storage instead of as originals folder.

The originals folder contains your original photo and video files. Database, config, cache, backup, thumbnail and sidecar files are saved in the storage folder.

- https://docs.photoprism.app/getting-started/docker/#photoprismoriginals

If you are new to docker we recommend setting up PhotoPrism using docker compose:
- https://docs.photoprism.app/getting-started/docker-compose/

**Reply by Nutztoyou** [reply]
Thank you, Thank you, Thank you, for pointing out the what should have been obvious to me, the amount of hours I've spent trying to find the error. 

I've now changed from 

-v /volume1/pdrivecopy/photos:/photoprism/storage 
to 
-v /volume1/pdrivecopy/photos:/photoprism/originals. 

Indexing has commenced. I've noticed that the database, config, cache, backup, thumbnail and sidecar files/folders are now saved in the originals folder. Is it possible to point PhotoPrism to keep all those files and folders in the docker folder? Reason being the originals folder is sync'd to pCloud so I'd like to keep those config files together to be backed up by the NAS.

If you can suggest a change to the task that I can re-run I will be very grateful.

I had a look at the links you suggested. There was lots of information and I feel is beyond me to configure.  If there is a simple change to me existing script, I feel I could manage.

Thanks again

**Reply by lastzero** [reply]
You can move the files manually. For this please check your configuration based on the documentation we provide, eg by running the "photoprism config" command and make sure you have volumes mounted for both the originals and storage folder.

**Comment by Nutztoyou** [comment]
Thanks guys. Everything seems to be working how I want it now. Cheers
